import-core/import-services/src/main/java/org/devgateway/importtool/services/ImportService.java

```diff
@@ -153,15 +157,18 @@ public IDestinationProcessor getDestinationProcessor(String processorName, Strin
 				getName().equals(processorName)).findFirst();
 
 		if (optional.isPresent()) {
-			try {
-				Constructor<?> c = Class.forName(optional.get().getDestinationProcessor().getClassName()).getDeclaredConstructor(String.class);
-				c.setAccessible(true);
-				processor = (IDestinationProcessor) c.newInstance(new Object[]{authenticationToken});
-				processor.setProcessorVersion(processorVersion);
-			} catch (InstantiationException | IllegalAccessException | ClassNotFoundException | NoSuchMethodException | SecurityException | IllegalArgumentException | InvocationTargetException e) {
+			/*try {*/
```

Comment:
Why is this commented out and not removed altogether?

Reply:
Done!